hw/ip/aes/pre_sca/alma/verify_aes.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# Script to formally verify the masking of the AES unit using Alma.
```

Comment:
Would it make sense to add a `set -e` here to stop early if something goes awry?

Reply:
Good idea, done.